### ISSUE TITLE
SVG is null

### ISSUE THREAD

**Issue by robbieh** [jdbranham/grafana-diagram#150]
I frequently get "SVG is null" while using this plugin. So much so that I have not been able to successfully create even one diagram yet. I'm at a complete loss for what to do since this happens even with a new panel (with the default graph) sometimes.

The server log doesn't show anything. The javscript console says:

```
Panel data error: TypeError: "svg is null"
    render mermaidAPI.js:378
    renderDiagram diagramControl.js:347
    updateDiagram diagramControl.js:370
    onDataReceived diagramControl.js:171
    emit index.js:120
    emit actions.ts:46
    handleQueryResult metrics_panel_ctrl.ts:239
    Angular 7
    jQuery 2
metrics_panel_ctrl.ts:178:11
```

**Comment by jdbranham** [comment]
Hi! I'll take a look at this today

**Comment by robbieh** [comment]
Awesome. I'll be online a lot today. If there is anything else I can provide or try, let me know.

**Comment by jdbranham** [comment]
@robbieh - I've committed a change that adds some error handling. If you still have trouble, paste your panel json here and I'll try to reproduce.

**Comment by robbieh** [comment]
I pulled 1.6.2 and started working with the plugin again. It was giving me the same behavior as before. And then I noticed that NoScript was showing something blocked. I allowed `grafana.com` and then the plugin started working. Even when I make errors in the Mermaid block, it keeps working. Strangely NoScript doesn't show that `grafana.com` is being sent requests anymore, and the plugin continues to work.

Does that make any sense? :D
